openblas/src/main/java/org/bytedeco/openblas/presets/openblas_nolapack.java

```diff
@@ -136,6 +136,22 @@ public class openblas_nolapack implements LoadEnabled, InfoMapper {
                .put(new Info("openblas_complex_float", "lapack_complex_float").cast().pointerTypes("FloatPointer", "FloatBuffer", "float[]"))
                .put(new Info("openblas_complex_double", "lapack_complex_double").cast().pointerTypes("DoublePointer", "DoubleBuffer", "double[]"));
 
+        String[] brokenFunctions = {
+            // not implemented by MKL
+            "cgesvdq", "dgesvdq", "sgesvdq", "zgesvdq", "clangb", "dlangb", "slangb", "zlangb",
+            "ctrsyl3", "dtrsyl3", "strsyl3", "ztrsyl3",
+            "sgedmd", "dgedmd", "cgedmd", "zgedmd", "sgedmdq", "dgedmdq", "cgedmdq", "zgedmdq",
+            // deprecated
+            "cggsvd", "dggsvd", "sggsvd", "zggsvd", "zggsvp", "cggsvp", "dggsvp", "sggsvp",
+            // extended
+            "cgbrfsx", "cporfsx", "dgerfsx", "sgbrfsx", "ssyrfsx", "zherfsx", "cgerfsx", "csyrfsx", "dporfsx", "sgerfsx", "zgbrfsx", "zporfsx",
+            "cherfsx", "dgbrfsx", "dsyrfsx", "sporfsx", "zgerfsx", "zsyrfsx", "cgbsvxx", "cposvxx", "dgesvxx", "sgbsvxx", "ssysvxx", "zhesvxx",
+            "cgesvxx", "csysvxx", "dposvxx", "sgesvxx", "zgbsvxx", "zposvxx", "chesvxx", "dgbsvxx", "dsysvxx", "sposvxx", "zgesvxx", "zsysvxx"};
+
```

Comment:
Most of those functions are probably available

Reply:
They are available only with the LAPACKE_ prefix, so I enabled these versions.

Reply:
These functions fail on android. Should we exclude them completely (as it was) or there is a way to make this part of the generator conditional on the operating system?
I'm fine with either, but you'll have to decide what is best aligned with javacpp-presets goals.

Reply:
It's not like they are not failing on Linux or Mac either, these functions do not exist at all. Linux and Mac simply allow to link against undefined functions, but Android and Windows do not, that's all. There is no reason to leave them there though, so please remove them.

Reply:
I've done this, but the error log identified a handful of other troublemakers from the "extended" sections, so I skipped them too. Fingers crossed...